functions-methods.js

```diff
@@ -33,4 +51,15 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+    function checkEmailValidity(str) {
+        const eRegex = RegExp("@");
+        const cRegex = RegExp(",");
+        if(eRegex.test(str) === true && cRegex.test(str) === false && str.charAt(str.length - 1) !== ".") {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
Hier zou je de uitkomst direct terug kunnen geven
```suggestion
        return eRegex.test(str) && !cRegex.test(str) && str.charAt(str.length - 1) !== ".";
```